cypress/support/Helpers.js

```diff
@@ -26,7 +26,7 @@ export function StringArrayFromElementText(selector, retainMarkup = false) {
   ConLog(`StringArrayFromElementText(${selector})`, elements.length)
   let returnValues = []
   for (let i = 0; i < elements.length; i++)  {
-    let text = retainMarkup ? elements[i].innerHTML : elements[i].innerText
```

Comment:
Been meaning to ask why do we use `innerText` instead of `textContent`?

> As a result, innerText is much more performance-heavy: it requires layout information to return the result.

> innerText will turn <br> elements into newline characters, while textContent will just ignore them.

In most cases it seems like we don't care about the layout and only care that the content in the node is the correct content.  I would think this could improve the speed and remove the need for code like this.

Reply:
Because I was unaware of `textContent`...so thanks for pointing that out.